region/brinstar/blue.json

```diff
@@ -244,19 +170,54 @@
         {
           "from": 1,
           "to": [
+            {
+              "id": 1,
+              "strats": [
+                {
+                  "name": "Leave with Runway",
+                  "notable": false,
+                  "requires": [],
+                  "exitCondition": {
+                    "leaveWithRunway": {
+                      "length": 4,
+                      "openEnd": 1
+                    }
+                  }
+                },
+                {
+                  "name": "Leave Charged",
+                  "notable": false,
+                  "requires": [
+                    {"obstaclesCleared": ["C"]},
+                    "canShinechargeMovement",
+                    {"canShineCharge": {
+                      "usedTiles": 25,
+                      "openEnd": 0
+                    }}
+                  ],
+                  "exitCondition": {
+                    "leaveCharged": {
+                      "framesRemaining": 80
+                    }
+                  },
+                  "devNote": "Destroy obstacle C by travelling between nodes 1 and 6."
+                }
+              ]
+            },
             {
               "id": 5,
               "strats": [
                 {
                   "name": "Room Entry Speedball",
                   "notable": false,
+                  "entranceCondition": {
+                    "comeInCharging": {
+                      "length": 2,
+                      "openEnd": 1
```

Comment:
The full runway here is length 3 with an open end. So the length of 2 here already has 1 subtracted, corresponding to the "unusableTiles: 1". I think this is okay, since it looks like you wouldn't hit the hopper until almost at the end of the last tile (which isn't being counted as part of the runway). If it sounds good, I could add a devNote to make it more clear about the 1 being subtracted and the reason why?

Reply:
The openEnd should be 0, from how I see it.  Since you cannot use it as part of shinecharge.

Reply:
Sure, changed it to 0 now. To be clear, this has nothing to do with the open end at the end of the platform, since with length 2 that last tile already wasn't considered part of the runway. Changing it to 0 just makes the usable runway even shorter (by 9 pixels), indicating that Samus needs to get blue before moving into the space above that last platform tile at all.